scripts/exglobal_makeprepbufr.sh

```diff
@@ -571,26 +577,26 @@ if [[ "$MAKE_NSSTBUFR" == 'YES' ]]; then
          if [ -s $file ]; then
               cat $file >> nsstbufr
               err_cat=$?
-            if [ $err_cat -ne 0 ]; then
+           if [ $err_cat -ne 0 ]; then
               msg="**WARNING: exit status $err_cat from cat of $file to nsstbufr"
               $DATA/postmsg "$jlogfile" "$msg"
-            fi
+           fi
          else
             echo $file is empty or does not exist
          fi # [ -s $file ]
 
-      done # for type in $DTYPS_nsst 
+      done # for type in $DTYPS_nsst
 
-      cp nsstbufr $COMOUT/${RUN}.${cycle}.nsstbufr
-      chgrp rstprod $COMOUT/${RUN}.${cycle}.nsstbufr
-      chmod 640 $COMOUT/${RUN}.${cycle}.nsstbufr
+      chmod 640 nsstbufr
       msg="NOTE: nsstbufr file contains RESTRICTED data, only users in \
```

Comment:
So this message isn't necessarily true now. It was true when `chgrp rstprod` was forced but that has been put within an if-block checking `CHGRP_RSTPROD` up above. Perhaps wrap this message in a check of `CHGRP_RSTPROD=YES` as well?

@ilianagenkova thoughts?

Reply:
Kate is right and her suggestion "Perhaps wrap this message in a check of CHGRP_RSTPROD=YES as well" is good.